### ISSUE TITLE
Some ideas i got

### ISSUE THREAD

**Issue by Marchin** [franksouza183/EvolvereSuit#78]
1-Evolvere mouse theme
2-A circle icon variant. Even though i wouldn't use them i see a lot of people like circle icons.
3-Porting Evolvere to Android.

**Comment by franksouza183** [comment]
Absolutely. Could open each idea on different issues? I believe the cursor theme is the closest to rise, then the circle variant, and finally the android version.

**Comment by MichaelTiebesl** [comment]
What i think is to leave the mouse theme and the circle icon variant. There are already great mouse themes. Circle icon variant is too much work and Evolvere is a beautiful square icon theme. Better concentrate on the new plasma5.  Create look and feel packages with grub, splash and SDDM. Make a few nice wallpapers also. It's nice to have a totall package. Just my idea.

**Comment by Marchin** [comment]
Sure! Now i open them

**Comment by franksouza183** [comment]
I'm already working on plasma theme, but must first begin by KDE4. If you want to test, remember: this is a hyper mega super beta version: https://github.com/franksouza183/EvolvereSuit/tree/master/kde-plasma-themes/kde4/EvolvereLight. @MichaelTiebesl you're right. I meant after the requests already made, the next step will be the cursor theme, next the circle icons variant, and finally the icon theme for android. Too much for one person, I need a team :tired_face:

**Comment by MichaelTiebesl** [comment]
Sorry Frank i dont have KDE4 anymore. I'm fully switch to plasma5 now. I test the new plasma already since it is out.  I'm already happy to see a beginning with it. Keep up the great work. If i can help i do it for sure.
